tests/membership.py

```diff
@@ -302,31 +302,6 @@ def recovery_shares_scenario(args):
         )
 
 
-@reqs.description("Send an unsigned request where signature is required")
```

Comment:
This seemed like a useful test, why is it going?

Reply:
This is a duplicate, and never called - there's another copy in `memberclient` which _is_ used.